asv_setup/config/robots/ASV.yaml

```diff
@@ -0,0 +1,13 @@
+failsafe:
+  publishers:
+    failSafeStatus: '/failSafeStatus'
+    hardwareOperationMode: '/hardWareOperationMode'
+  subscribers:
+    softWareKillSwitch: '/softWareKillSwitch'
+    softWareOperationMode: '/softWareOperationMode'
+
+  gpio:
```

Comment:
Looks clean. Here replace the pin numbers for killswitch and operation with the new pins that @Benjamin-A has provided :)
ie: read here: https://vortex.a2hosted.com/index.php/ASV_Fail_Safe#Raspberry_Pi_4B_Connections_(SW)